scripts/create-component/plop-templates/src/components/{{componentName}}/{{componentName}}.types.ts.hbs

```diff
@@ -4,16 +4,15 @@ export type {{componentName}}Slots = {
   root: Slot<'div'>;
 };
 
-type {{componentName}}Commons = {
-  // TODO Add things shared between props and state here
-};
-
 /**
  * {{componentName}} Props
  */
-export type {{componentName}}Props = ComponentProps<{{componentName}}Slots> & {{componentName}}Commons;
+export type {{componentName}}Props = ComponentProps<{{componentName}}Slots> & {};
```

Comment:
Nit question: Does adding the empty object do anything?

Reply:
Doesn't do anything, it just adds a place for people to start writing their own props.